net/openthread-br/Makefile

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2022 Stijn Tintel <stijn@linux-ipv6.be>
+# SPDX-License-Identifier: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openthread-br
+PKG_SOURCE_DATE:=2023-03-29
+PKG_SOURCE_VERSION:=cbeaf817c54ad06d0238d0fa2ec2a5b4b32ba805
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/openthread/ot-br-posix.git
+PKG_MIRROR_HASH:=b0968851538783dd6580be85e11257ceff5fe5ce8e5e60576cc7a05cde770524
+
+#PKG_BUILD_DIR:=$(BUILD_DIR)/ot-br-posix-thread-br-certified-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Stijn Tintel <stijn@linux-ipv6.be>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/openthread-br
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=OpenThread Border Router
+  DEPENDS:= \
+	+libblobmsg-json \
+	+libjson-c \
+	+libncurses \
+	+libreadline \
+	+libstdcpp \
+	+libubox \
+	+libubus \
+	+mdnsd \
+	+mdnsresponder
+endef
+
+define Package/openthread-br/description
+  A Thread border router for POSIX-based platforms.
+endef
+
+define Package/openthread-br/conffiles
+/etc/config/openthread
+/var/lib/thread
```

Comment:
I had to remove this to get it to compile on 21.02.7

Reply:
This is targeted at master. If you want to use it on anything else, you're on your own.